pkgs/development/python-modules/decorating/default.nix

```diff
@@ -0,0 +1,21 @@
+{ stdenv, buildPythonPackage, fetchPypi, isPy27, isPy34, isPy35, isPy36 }:
+
+assert (isPy27 || isPy34 || isPy35 || isPy36);
+
+buildPythonPackage rec {
+  name = "${pname}-${version}";
```

Comment:
The `name` attribute is added by `buildPython*` and should therefore be removed.